### PR TITLE
Update sysctl jinja macros for sysctl.d locations

### LINKED ISSUE #2941: update describe_sysctl_option_value jinja macro for rhel7 sysctl locations
Template should refer users to add settings in ``/etc/sysctl.d`` vs ``/etc/sysctl.conf`` 

https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/modules/macros.jinja#L525#L532

`````
{{% macro describe_sysctl_option_value(sysctl, value) -%}}
    To set the runtime status of the <code>{{{ sysctl }}}</code> kernel parameter,
    run the following command:
    <pre>$ sudo sysctl -w {{{ sysctl }}}={{{ value }}}</pre>

    If this is not the system default value, add the following line to <tt>/etc/sysctl.conf</tt>:
    <pre>{{{ sysctl }}} = {{{ value }}}</pre>
{{%- endmacro %}}
`````

Thanks to Dan White for pointing this out.

### PR DESCRIPTION
**Pull request by redhatrises**
#### Description:
- Update sysctl jinja macros for sysctl.d locations

- Fixes #2941

### CODE REVIEW

File: shared/macros.jinja [hunk 1]
```diff
@@ -513,7 +513,11 @@ ocil_clause: "{{{ sebool }}} is not enabled"
     <pre>$ sysctl {{{ sysctl }}}</pre>
     The output of the command should indicate a value of <code>{{{ value }}}</code>.
     If this value is not the default value, investigate how it could have been
-    adjusted at runtime, and verify it is not set improperly in <tt>/etc/sysctl.conf</tt>.
+    adjusted at runtime, and verify it is not set improperly by running the
+    following command to search for such lines in all files in <tt>/etc/sysctl.d</tt>
+    and the deprecated <code>/etc/sysctl.conf</code>:
+    <pre>$ grep -r {{{ module }}} /etc/sysctl.conf /etc/sysctl.d</pre>
```
Comment (yuumasato) [inline shared/macros.jinja]: Where is `module` defined?

Comment (redhatrises) [reply shared/macros.jinja]: haha... oops... fixed

File: shared/macros.jinja [hunk 2]
```diff
@@ -513,7 +513,11 @@ ocil_clause: "{{{ sebool }}} is not enabled"
     <pre>$ sysctl {{{ sysctl }}}</pre>
     The output of the command should indicate a value of <code>{{{ value }}}</code>.
     If this value is not the default value, investigate how it could have been
-    adjusted at runtime, and verify it is not set improperly in <tt>/etc/sysctl.conf</tt>.
+    adjusted at runtime, and verify it is not set improperly by running the
+    following command to search for such lines in all files in <tt>/etc/sysctl.d</tt>
+    and the deprecated <code>/etc/sysctl.conf</code>:
+    <pre>$ grep -r {{{ sysctl }}} /etc/sysctl.conf /etc/sysctl.d</pre>
+
```
Comment (dahaic) [inline shared/macros.jinja]: Sorry for being nitpicky, but this is quite hard to follow sentence. Can you rephrase it? For example
"...adjusted at runtime, and verify it is not set improperly. This has to be checked in all files in <tt>/etc/sysctl.d</tt> and the deprecated <code>/etc/sysctl.conf</code>. Following command will do just that:"

Comment (redhatrises) [reply shared/macros.jinja]: sure.... done.

**Comment by dahaic** [comment]
Thank you, ACK
